src/devices/Ht16k33/Command.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Iot.Device.Ht16k33
+{
+    internal class Command
```

Comment:
should be an enum

Reply:
Thought I could spare a few casts, but done 🙂